tests/unit/test_graph_selector_methods.py

```diff
@@ -586,6 +588,24 @@ def test_select_saved_query_by_tag(manifest: Manifest) -> None:
     search_manifest_using_method(manifest, method, "any_tag")
 
 
+def test_select_unit_test(manifest: Manifest) -> None:
+    test_model = make_model("test", "my_model", "select 1 as id")
+    unit_test = make_unit_test("test", "my_unit_test", test_model)
+    manifest.unit_tests[unit_test.unique_id] = unit_test
```

Comment:
Do you feel like this is easier to use or we somehow have a fixture called `unittests` that will be required by manifest and you return all unittest nodes in it easier? Or maybe not so much difference?
cc @QMalcolm

Reply:
I thought this was very easy to use! and keeps unit tests more narrow and isolated from one another in their setup